core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -214,6 +214,14 @@ case class WorkflowQueueStatusResponse
   workflowCountsByStatus: Map[String, Int]
 )
 
+case class WorkflowQueueStatusByUserResponse
+(
+  statuses: Map[String, Int],
```

Comment:
Can we type alias this as `StatusCounts`, and _possibly_ do something similar for `users`? `users.convertTo[Map[String, Map[String, Map[String, Int]]]]` on 336 below is a bit of a mindbender.